codex/full/deploy.yml

```diff
@@ -0,0 +1,223 @@
+---
+version: "2.0"
+
+endpoints: 
+  nodeip2:
+    kind: ip
+
+services:
+  codex:
+    image: codexstorage/nim-codex:0.2.0
+    expose:
+    - port: 8070
+      as: 8070
+      proto: tcp
+      to:
+        - global: true
+          ip: nodeip2
+    - port: 8090
+      as: 8090
+      proto: udp
+      to:
+        - global: true
+          ip: nodeip2
+    - port: 8080
+      as: 8080
+      to:
+        - service: nginx
+    env:
+      - DATA_DIR=/codex/data
+      - ETH_PRIVATE_KEY=
+      - PUBLIC_IP=
+      - CODEX_LOG_LEVEL=debug
+      - CODEX_BLOCK_TTL=30d
+    command:
+    - sh
+    - -c
+    args:
+    - >
+        if [ -z "${PUBLIC_IP}" ]; then
+          echo "Missing \$PUBLIC_IP=public_ip_of_the_deplyoment"
+          sleep 120
+          exit 1
+        fi
+
+        mkdir -p ${DATA_DIR}
+        chmod 0700 ${DATA_DIR}
+
+        quota=$(df --output=size --block-size=1 /codex | tail -1)
+
+        BOOTSTRAP_NODES=$(for i in $(curl https://spr.codex.storage/testnet 2> /dev/null); do echo --bootstrap-node=${i}; done)
+
+        exec /docker-entrypoint.sh codex \
+          --storage-quota=${quota} \
+          --data-dir=${DATA_DIR} \
+          --api-port=8080 \
+          --api-bindaddr=0.0.0.0 \
+          --disc-port=8090 \
+          --listen-addrs=/ip4/0.0.0.0/tcp/8070 \
+          --nat=extip:${PUBLIC_IP}\
+          ${BOOTSTRAP_NODES}\
+          persistence \
+          --eth-provider=https://rpc.testnet.codex.storage
+
+    params:
+      storage:
+        codex:
+          mount: /codex
+          readOnly: false
+  codexapp:
+    image: codexstorage/codex-marketplace-ui:sha-4b2e179
+    expose:
+    - port: 80
+      as: 80
+      to:
+        - service: nginx
+  nginx:
+    image: nginx:1.27
+    expose:
+      - port: 80
+        as: 80
+        to:
+          - global: true
+            ip: nodeip2
+      - port: 443
+        as: 443
+        to:
+          - global: true
+            ip: nodeip2
+    env:
+      - BASIC_AUTH_USER=codex
+      - BASIC_AUTH_PASS=passsecret
+      - DOMAIN=
```

Comment:
I don't know what this does :o

Reply:
Read the README, maybe?:-P https://github.com/akash-network/awesome-akash/pull/639/files#diff-2cf623688b5f92a6e188d2c89a442e6dff984055c537c0f8d01bd03f035f5917R27

Reply:
Reading READMEs? what do you think I am, a normal person???
Yeah ok yes, good point. Makes sense. 👍